database/test_database.go

```diff
@@ -95,7 +95,7 @@ func TestSimpleKeyValue(t *testing.T, db Database) {
 	require.False(has)
 
 	_, err = db.Get(key)
-	require.Equal(ErrNotFound, err)
+	require.ErrorIs(err, ErrNotFound)
```

Comment:
this doesn't fail the linter (and I don't think there's a regex we can add to consistently test for this). Just need to be more vigilant going forward